plugin/evm/vm.go

```diff
@@ -272,6 +272,13 @@ func (vm *VM) Initialize(
 	vm.metadataDB = prefixdb.New(metadataPrefix, vm.db)
 	vm.warpDB = prefixdb.New(warpPrefix, vm.db)
 
+	if vm.config.PruneWarpDB {
+		err := database.ClearPrefix(vm.db, vm.db, warpPrefix)
+		if err != nil {
+			return fmt.Errorf("failed to prune warpDb")
+		}
+	}
```

Comment:
Will this clear the correct data passing in `warpPrefix` to `database.ClearPrefix` to clear the database created by `prefixdb.New(warpPrefix, ...)` ?

Reply:
Sounds good to me!